README.rst

```diff
@@ -588,6 +588,31 @@ which is indexed in degrees, the 'distance' will be interpreted as degrees. Set
 to 'True' in order to convert an input distance in meters to degrees. This conversion is approximate, and the errors
 at latitudes > 60 degrees are > 25%.
 
+DistanceToPointOrderingFilter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Provides a ``DistanceToPointOrderingFilter``, **available on Django >= 3.0**, which is a subclass of ``DistanceToPointFilter``.
+Orders a queryset by distance to a given point.
+
+``views.py:``
+
+.. code-block:: python
+
+    from rest_framework_gis.filters import DistanceToPointOrderingFilter
+
+    class LocationList(ListAPIView):
+
+        queryset = models.Location.objects.all()
+        serializer_class = serializers.LocationSerializer
+        distance_ordering_filter_field = 'geometry'
+        filter_backends = (DistanceToPointOrderingFilter, )
+
+We can then order the results by passing a point in (lon, lat) format in the URL.
+
+eg:.
+``/location/?point=-122.4862,37.7694&format=json``
+will order the results by the distance to the point (-122.4862, 37.7694).
```

Comment:
Sorry another thing I realized now, are the points ordered from the nearest to the most distant or the opposite?
Can we make this clearer please?

What if a user wants to reverse the order? Would be asking too much to add an optional way to do that?
Eg: 

```
# ascending order
/location/?point=-122.4862,37.7694&direction=asc&format=json

# descending order
/location/?point=-122.4862,37.7694&direction=desc&format=json
```